transport/src/main/java/io/camunda/zeebe/transport/stream/impl/RemoteStreamTransport.java

```diff
@@ -15,16 +15,14 @@
 import io.camunda.zeebe.transport.stream.impl.messages.RemoveStreamRequest;
 import io.camunda.zeebe.transport.stream.impl.messages.StreamTopics;
 import io.camunda.zeebe.util.buffer.BufferReader;
-import io.camunda.zeebe.util.buffer.BufferUtil;
 import java.time.Duration;
 import java.util.concurrent.CompletableFuture;
 import java.util.function.Function;
-import org.agrona.concurrent.UnsafeBuffer;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public final class RemoteStreamTransport<M extends BufferReader> extends Actor {
-  private static final byte[] EMPTY_RESPONSE = new byte[0];
+  private static final byte[] EMPTY_REQUEST = new byte[0];
```

Comment:
:wrench: I guess since we use it for both request and response, we can just name it `EMPTY_PAYLOAD` or something. It's a bit weird to return something called `EMPTY_REQUEST` as a response :smile: